core/src/main/java/io/questdb/cairo/wal/MetadataService.java

```diff
@@ -100,6 +102,8 @@ void addColumn(
 
     void renameTable(@NotNull CharSequence fromNameTable, @NotNull CharSequence toTableName);
 
+    void setDeduplicationStatus(boolean status, LongList columnsIndexes, int startIndex, long count);
```

Comment:
should split this method into two:

Reply:
split into

```
void enableDeduplicationWithUpsertKeys(LongList columnsIndexes);

void disableDeduplication();
```

in https://github.com/questdb/questdb/pull/3524/commits/8699ee84624f26e3c01491745fc4ef3fd3e90633